### PR TITLE
Align x

### PR DESCRIPTION
**Pull request by gmischler**
- [x] The GitHub pipeline is OK (green), 
      meaning that both `pylint` (static code analyzer) and `black` (code formatter) are happy with the changes of this PR.
- [x] A unit test is covering the code added / modified by this PR
- [x] This PR is ready to be merged 
- [x] In case of a new feature, docstrings have been added, with also some documentation in the `docs/` folder
- [x] A mention of the change is present in `CHANGELOG.md`
- [x] The PR description or comment contains a picture of a cute animal (not mandatory but encouraged 🦑 )

Inspired by the misunderstanding in #411 about horizontal and vertical centering...

The new alignment type "X" for relevant text functions aligns text horizontally around the current x position.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ in order to get warned about deprecated features used in your code.
 This can also be enabled programmatically with `warnings.simplefilter('default', DeprecationWarning)`.
 
 ## [2.5.5] - not released yet
+### Added
+- A new option `align="X"` allows to center text around the current x position, thanks to @gmischler
```
Comment (Lucas-C) [inline CHANGELOG.md:21]: Could you specify what methods accept this new option please?

**Review by Lucas-C — APPROVED** [review]
This is a very useful addition!
Thank you @gmischler \o/
About naming: why `X` and not e.g. `M` for MIDDLE?

File: fpdf/enums.py [hunk 2]
```diff
@@ -103,6 +103,9 @@ class Align(CoerciveEnum):
     C = intern("CENTER")
     "Center text"
 
+    X = intern("X_CENTER")
+    "Center text around current x"
```
Comment (Lucas-C) [inline fpdf/enums.py:107]:
```suggestion
    "Center text horizontally around current x position"
```

File: fpdf/enums.py [hunk 3]
```diff
@@ -103,6 +103,9 @@ class Align(CoerciveEnum):
     C = intern("CENTER")
     "Center text"
```
Comment (Lucas-C) [inline fpdf/enums.py:104]:
```suggestion
    "Center text horizontally"
```

File: CHANGELOG.md (hunk 1 above)
Comment (gmischler) [reply CHANGELOG.md:21]: done

**Comment by gmischler** [comment]
> About naming: why `X` and not e.g. `M` for MIDDLE?

As explained in the vertical alignment discussion: I'd prefer to follow the HTML/CSS terminology of "center" for horizontal and "middle" for vertical alignment, to reduce any potential confusion.
Apart from that,  the "X" can also be seen as a visual representation of the centerpoint. ❌

**Comment by Lucas-C** [comment]
Merged! Thank you @gmischler 😊

**Pull request merged**